src/App.js

```diff
@@ -1,9 +1,18 @@
+import Header from './components/Header';
+import BodyContainer from './components/BodyContainer';
+import './App.css';
+import members_data from './data.json'
+
+console.log(members_data,"Array")
 
 function App() {
   return (
-    <div className="App">
-      <h1>Master branch</h1>
-    </div>
+    <>
+      <Header/>
+      <div className="box">
```

Comment:
you can add the div inside BodyContainer. Does this have any special role here?

Reply:
No.Will correct in assignment 5